pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -789,15 +790,16 @@ class Build {
         context.node('master') {
             context.stage("sign installer") {
                 try {
-                    if (buildConfig.TARGET_OS == "windows") {
+                    if (buildConfig.TARGET_OS == "mac" || buildConfig.TARGET_OS == "windows") {
```

Comment:
Why are we adding `mac` here?

Reply:
To keep in sync with the upstream. This function is not called for Semeru builds on win and mac, only for aix due to: https://github.com/ibmruntimes/ci-jenkins-pipelines/blob/d138bf0e89da638cbb386dffa3c1ebafd46bcfaf/pipelines/build/common/openjdk_build_pipeline.groovy#L1886